source/Nevermore/Querying/AST/Select.cs

```diff
@@ -7,14 +7,16 @@ public class Select : ISelect
         readonly ISelectSource from;
         readonly Where where;
         readonly OrderBy orderBy; // Can be null
+        readonly GroupBy groupBy; // Can be null
 
-        public Select(IRowSelection rowSelection, ISelectColumns columns, ISelectSource from, Where where, OrderBy orderBy)
+        public Select(IRowSelection rowSelection, ISelectColumns columns, ISelectSource from, Where where, GroupBy groupBy, OrderBy orderBy)
```

Comment:
This class and its friends are public - does that mean they're part of our public API for nevermore? 

If so, as is this would be a breaking change, so we'll either need to add backwards compat, or do a major version bump.